pkgs/tools/misc/coreutils/default.nix

```diff
@@ -124,6 +132,8 @@ stdenv.mkDerivation rec {
     [
       perl
       xz.bin
+      autoconf
+      automake
```

Comment:
I tried moving autoreconfHook here, but that didn't work. The patch touches one of the tests .mk files.

Dies with:

```
m4/yield.m4:9: gl_YIELD is expanded from...
m4/gnulib-comp.m4:1139: gl_INIT is expanded from...
configure.ac:63: the top level
configure:9985: error: possibly undefined macro: gl_ANYTHREADLIB_EARLY
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
configure:23465: error: possibly undefined macro: gl_PTHREADLIB
configure:23578: error: possibly undefined macro: gl_WEAK_SYMBOLS
configure:26833: error: possibly undefined macro: gl_TYPE_WINT_T_PREREQ
```